src/http2_connection.erl

```diff
@@ -69,14 +73,15 @@
           end_headers = false       :: boolean()
 }).
 
-
 -record(stream, {
-          id :: stream_id(),
-          pid :: pid(),
-          send_window_size :: non_neg_integer(),
-          recv_window_size :: non_neg_integer(),
-          queued_data :: undefined | done | binary(),
-          body_complete = false :: boolean()
+          id                    :: stream_id(),
+          pid                   :: undefined | pid(),
+          send_window_size      :: non_neg_integer(),
+          recv_window_size      :: non_neg_integer(),
+          queued_data           :: undefined | done | binary(),
+          body_complete = false :: boolean(),
+          response_headers      :: undefined | hpack:headers(),
```

Comment:
It's part of the tombstone. The stream is saying "Hey connection, I'd like to die now, but I can't because maybe somebody will come along and and ask what the response was. If you knew what that was, it could be your job and I could go die"